language/Portuguese_Brazil/strings.po

```diff
@@ -20,871 +20,871 @@ msgstr ""
 
 #: src/emu/ui/auditmenu.cpp:111
 msgid "Audit in progress..."
-msgstr ""
+msgstr "Auditoria em progresso"
 
 #: src/emu/ui/barcode.cpp:72
 msgid "New Barcode:"
-msgstr ""
+msgstr "Novo c�digo de barras"
```

Comment:
you missed the ":" in the end of the string. Please try to keep the same punctiation and case-style in translations.